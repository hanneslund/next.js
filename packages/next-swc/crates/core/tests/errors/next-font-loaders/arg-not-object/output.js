import s from "@next/font/google?Sacramento;{}";
import s from "@next/font/google?Sacramento;{}";
const obj = {};
