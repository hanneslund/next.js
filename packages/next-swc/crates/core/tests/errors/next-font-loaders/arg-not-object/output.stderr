
  x Font loader must be called with an options object
   ,-[input.js:3:1]
 3 | const s = Sacramento(obj)
   :           ^^^^^^^^^^^^^^^
   `----

  x Font loader must be called with an options object
   ,-[input.js:4:1]
 4 | const s = Sacramento(10)
   :           ^^^^^^^^^^^^^^
   `----
